PoGo.NecroBot.Logic/State/VersionCheckState.cs

```diff
@@ -29,6 +29,9 @@ public class VersionCheckState : IState
         public const string RemoteReleaseUrl =
             "https://github.com/Necrobot-Private/NecroBot/releases/download/v";
 
+        public const string ChangelogUri =
+             "https://cdn.rawgit.com/Necrobot-Private/NecroBot/master/CHANGELOG.md";
```

Comment:
If it's erroring on yours, I'd recommend changing this to raw.githubusercontent.com as although it doesn't error for most, it'll ensure max compatibility